testlapack/dgecon.go

```diff
@@ -44,6 +44,20 @@ func DgeconTest(t *testing.T, impl Dgeconer) {
 			condOne: 0.024740155174938,
 			condInf: 0.012034465570035,
 		},
+		/*
+			// Dgecon does not match Dpocon for this case. https://github.com/xianyi/OpenBLAS/issues/664.
+			{
```

Comment:
The comment for this could be more informative. Same for the one in dpocon, though that one makes more sense; at first glance it is difficult to understand why dpocon is relevant to the dgecon test. It would be good to have a bit more explanation about what is expected and what is wrong.

Am I right that dgecon is wrong here since the inf-norm doesn't match the 1-norm and the matrix is symmetric?